### PR TITLE
made the part about server_name better readable

### PR DESCRIPTION
**Pull request by missytake**
We received this feedback a lot during our OFFDEM workshop.

Anything else to fix?

### CODE REVIEW

**Comment by missytake** [comment]
These changes should be tried out before they can be merged - best on Saturday.

**Review by hpk42 — COMMENTED** [review]
lgtm, thanks!

File: en/serverguide.md [hunk 1]
```diff
@@ -293,13 +299,61 @@ You can setup ufw as a firewall to protect mailadm against DoS attacks:
...
+you can disable POP3.
+
+To do this, add the following to `mailcow.conf`:
+
+```
+POP_PORT=127.0.0.1:110
+POPS_PORT=127.0.0.1:995
+```
+
+Then apply the changes with `sudo docker compose up -d`.
```
Comment (ralphtheninja) [inline en/serverguide.md:307]: Maybe use `sudo docker compose restart` instead? Assuming the service has been started. I don't know if you can just restart a single service like below or if you need to restart everything.

Comment (missytake) [reply en/serverguide.md:307]: In this scenario, `docker compose up` will re-run all containers whose definition was changed by editing those variables. Which is pretty much exactly what we want. `docker compose restart` would just re-start all existing containers, maybe even without changing their configuration, it's not what we want here.

**Pull request merged**